<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QGroupBox" name="m_srcGrpBox" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="title" >
          <string>Template</string>
         </property>
         <property name="flat" >
          <bool>false</bool>
         </property>
         <property name="checkable" >
          <bool>false</bool>
         </property>
         <property name="checked" >
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <item>
           <widget class="View3Qt" name="m_templateView" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet" >
             <string notr="true" >background-color: rgb(0, 0, 0);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="View3Qt" name="m_templateSliceView" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet" >
             <string notr="true" >background-color: rgb(0, 0, 0);</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="m_trgtGrpBox" >
         <property name="title" >
          <string>Specimen</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2" >
          <item>
           <widget class="View3Qt" name="m_targetView" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet" >
             <string notr="true" >background-color: rgb(0, 0, 0);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="View3Qt" name="m_targetSliceView" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet" >
             <string notr="true" >background-color: rgb(0, 0, 0);</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="ViewTree" name="m_templateItemTree" >
         <property name="minimumSize" >
          <size>
           <width>191</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>191</width>
           <height>16777215</height>
          </size>
         </property>
         <column>
          <property name="text" >
           <string>Items</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>View</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Slice</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="ViewTree" name="m_targetItemTree" >
         <property name="minimumSize" >
          <size>
           <width>191</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>191</width>
           <height>16777215</height>
          </size>
         </property>
         <column>
          <property name="text" >
           <string>Items</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>View</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Slice</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout_4" >
     <item>
      <widget class="QSpinBox" name="m_frameSBox" >
       <property name="minimum" >
        <number>0</number>
       </property>
       <property name="maximum" >
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_tblStart" >
       <property name="toolTip" >
        <string>Display first tableau</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../../../Resources.qrc" >
         <normaloff>:/Images/startframe.png</normaloff>:/Images/startframe.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_tblLeftE" >
       <property name="toolTip" >
        <string>Display previous tableau x10</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../../../Resources.qrc" >
         <normaloff>:/Images/firstframe.png</normaloff>:/Images/firstframe.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_tblLeft" >
       <property name="toolTip" >
        <string>Display previous tableau</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../../../Resources.qrc" >
         <normaloff>:/Images/stepprev.png</normaloff>:/Images/stepprev.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_tblRight" >
       <property name="toolTip" >
        <string>Display next tableau</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../../../Resources.qrc" >
         <normaloff>:/Images/stepnext.png</normaloff>:/Images/stepnext.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_tblListRightE" >
       <property name="toolTip" >
        <string>Display next tableau x10</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../../../Resources.qrc" >
         <normaloff>:/Images/lastframe.png</normaloff>:/Images/lastframe.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_tblEnd" >
       <property name="toolTip" >
        <string>Display last tableau</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../../../Resources.qrc" >
         <normaloff>:/Images/endframe.png</normaloff>:/Images/endframe.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="m_filmShow" >
       <property name="text" >
        <string>+</string>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="m_filmGrpBox" >
     <property name="title" >
      <string>Filmstrip</string>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
     <property name="checkable" >
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QSlider" name="m_frameSlider" >
          <property name="minimum" >
           <number>0</number>
          </property>
          <property name="maximum" >
           <number>0</number>
          </property>
          <property name="pageStep" >
           <number>1</number>
          </property>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition" >
           <enum>QSlider::TicksBelow</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4" >
          <property name="spacing" >
           <number>2</number>
          </property>
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="m_frameLBL" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Frame:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="m_frameCountLBL" >
              <property name="minimumSize" >
               <size>
                <width>40</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="baseSize" >
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="lineWidth" >
               <number>1</number>
              </property>
              <property name="text" >
               <string>0</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="m_lblStepsPerFrame" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Steps/Frame:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_stepsFrameTXT" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string>10</string>
              </property>
              <property name="maxLength" >
               <number>3</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="m_lblSecPerFrame" >
              <property name="text" >
               <string>Secs/Frame:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_secFrameTXT" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize" >
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="inputMask" >
               <string/>
              </property>
              <property name="text" >
               <string>1.0</string>
              </property>
              <property name="maxLength" >
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5" >
            <property name="spacing" >
             <number>3</number>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="m_filmFirstFrame" >
              <property name="toolTip" >
               <string>Moves to previous frame</string>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../../../Resources.qrc" >
                <normaloff>:/Images/firstframe.png</normaloff>:/Images/firstframe.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_filmStepPrev" >
              <property name="toolTip" >
               <string>Moves to previous frame step</string>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../../../Resources.qrc" >
                <normaloff>:/Images/stepprev.png</normaloff>:/Images/stepprev.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_filmStepNext" >
              <property name="toolTip" >
               <string>Moves to next frame step</string>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../../../Resources.qrc" >
                <normaloff>:/Images/stepnext.png</normaloff>:/Images/stepnext.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_filmLastFrame" >
              <property name="toolTip" >
               <string>Moves to next frame</string>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../../../Resources.qrc" >
                <normaloff>:/Images/lastframe.png</normaloff>:/Images/lastframe.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>10</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="m_filmPlayBackward" >
              <property name="toolTip" >
               <string>Play backwards</string>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../../../Resources.qrc" >
                <normaloff>:/Images/playb.png</normaloff>:/Images/playb.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_filmPlay" >
              <property name="toolTip" >
               <string>Play forwards</string>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../../../Resources.qrc" >
                <normaloff>:/Images/play.png</normaloff>:/Images/play.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_filmPause" >
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../../../Resources.qrc" >
                <normaloff>:/Images/pause.png</normaloff>:/Images/pause.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_filmStop" >
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../../../Resources.qrc" >
                <normaloff>:/Images/stop.png</normaloff>:/Images/stop.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ViewTree</class>
   <extends>QTreeWidget</extends>
   <header>../ViewTree/ViewTree.h</header>
  </customwidget>
  <customwidget>
   <class>View3Qt</class>
   <extends>QGraphicsView</extends>
   <header>../Viewers/View3Qt.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>m_filmShow</sender>
   <signal>clicked(bool)</signal>
   <receiver>m_filmGrpBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>725</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel" >
     <x>372</x>
     <y>535</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
